FreeRTOS/FreeRTOS
# Set configMAX_SYSCALL_INTERRUPT_PRIORITY to 4 for QEMU demos

Set configMAX_SYSCALL_INTERRUPT_PRIORITY to 4 for QEMU demos

Description
-----------
Set configMAX_SYSCALL_INTERRUPT_PRIORITY to 4 instead of 5 to avoid hitting the assert 
`(configASSERT( ( configMAX_SYSCALL_INTERRUPT_PRIORITY & 0x1U ) == 0U ); )`
This is in relation to the [PR]()

Test Steps
-----------
Run these Demo on QEMU and the assert is hit.

Checklist:
----------



Related Issue
-----------
#984


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
